core/arch/x86/decode_table.c

```diff
@@ -6679,269 +6679,208 @@ const instr_info_t evex_W_extensions[][2] = {
   {    /* evex_W_ext 0 */
```

Comment:
In the commit title:

> Formatting, evex_W_extensions 

The comma is strange: s/,/of/ or sthg

Reply:
Fixed.